### ISSUE TITLE
Application and its namespace not getting displayed in mayactl volume describe command.

### ISSUE THREAD

**Issue by gprasath** [openebs/openebs#2405]
## Description
 `mayactl volume describe --volname <PV> has to describe the volume information, status and its usage.
It is observed that the application and its namespace are not displayed when it is deployed on different namespace.

```
bash-4.3# mayactl volume describe --volname pvc-fa04dddc-367a-11e9-af96-42010a800043 -n openebs

Portal Details :
----------------
IQN               :   iqn.2016-09.com.openebs.cstor:pvc-fa04dddc-367a-11e9-af96-42010a800043
Volume            :   pvc-fa04dddc-367a-11e9-af96-42010a800043
Portal            :   10.19.253.47:3260
Size              :   5G
Controller Status :   running,running,running
Controller Node   :   gke-e2e-cluster-1-default-pool-2b6b9443-5t2v
Replica Count     :   3

Application Details:
--------------------
Application Pod Name      : N/A
Application Pod Namespace : N/A

Replica Details :
-----------------
NAME                                                                STATUS      POOL NAME                  NODE
----                                                                ------      ---------                  -----
pvc-fa04dddc-367a-11e9-af96-42010a800043-cstor-sparse-pool-1amj     Running     cstor-sparse-pool-1amj     gke-e2e-cluster-1-default-pool-2b6b9443-5t2v
pvc-fa04dddc-367a-11e9-af96-42010a800043-cstor-sparse-pool-s8hb     Running     cstor-sparse-pool-s8hb     gke-e2e-cluster-1-default-pool-2b6b9443-c8wg
pvc-fa04dddc-367a-11e9-af96-42010a800043-cstor-sparse-pool-u1ju     Running     cstor-sparse-pool-u1ju     gke-e2e-cluster-1-default-pool-2b6b9443-f4sc
```

**Comment by kmova** [comment]
May need to revisit this issue. A fix was included and reverted between 0.8 and 0.8.2, as it was causing a regression with provisioning pods with multiple PVCs.

**Comment by vharsh** [comment]
This might end up getting removed, need to discuss with the team.
Ref: https://github.com/openebs/openebs/issues/2470
